src/material/datepicker/date-range-input-parts.ts

```diff
@@ -180,6 +191,13 @@ export class MatStartDate<D> extends _MatDateRangeInputBase<D>
     }
   }
 
+  protected _formatValue(value: D | null) {
```

Comment:
does `MatEndDate` need this too?

Reply:
Only `MatStartDate` needs it, because we need to ensure that the mirror value for the autosizing is always up to date. `MatEndDate` doesn't do autosizing.